### DISCUSSION TITLE
Enable/Disable random first selection in Song Select

### DISCUSSION THREAD

**Discussion by CrumblyLiquid** [ppy/osu#21082] [Ideas]
It would be great to add a switch for enabling/disabling random beatmap selection when you first open song selection. It's especially annoying with the `Last Played` category since it's purpose is picking from the top of the most recently played beatmaps, not random selection.

This would probably fall under the `User Interface` -> `Song Select` setting group.

**Comment by peppy** [comment]
Can agree with this. Not sure it needs to ever be random? Remembering the previous session's selection may work best in all cases.

Curious on other people's thoughts.

**Reply by peppy** [reply]
Polling this at https://twitter.com/ppy/status/1593090802888376321

**Reply by zoey-on-github** [reply]
as someone who plays a ton of beatmaps in one session, i prefer the random approach because i just sort of play a beatmap, then move onto the next, i don't really find myself wanting to stay on a beatmap after i'm done with it

**Comment by LolloDev5123** [comment]
It would also be cool if osu!lazer saved the last selected playlist, as I have a separate playlist for anime
